package.json

```diff
@@ -9,7 +9,7 @@
   "peerDependencies": {
     "eslint": "*"
   },
-  "author": "Chris Banks <github@christopherbradleybanks.com>",
+  "author": "Pajarraco <https://github.com/pajarraco>",
```

Comment:
Lets not change the author field of the package json
instead add "contributors": [{
  "name" : "Barney Rubble",
  "email" : "b@rubble.com",
  "url" : "http://barnyrubble.tumblr.com/"
}] to the package JSON

Reply:
Hi, I was running this 
'folders/match-regex': [2, /^(?!\s)(?![\s\S]*\s$)[a-zA-Z0-9\s()-_&\%\.\/]+$/g, '/src/'], 

and noting was matching the expression, then I found this 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test#using_test_on_a_regex_with_the_global_flag

here they explain what happens with the index on "regex.test('foo', 'g')".